scripts/exdump_monitor.sh

```diff
@@ -551,7 +551,7 @@ DTIM_latest_gpsipw=-0.01
 DTIM_earliest_ahicsr=-1.00
 DTIM_latest_ahicsr=-0.01
 
-$ushscript_dump/bufr_dump_obs.sh $dumptime 0.50 1 msonet gpsipw \
+$USHbufr_dump/bufr_dump_obs.sh $dumptime 0.50 1 msonet gpsipw \
```

Comment:
Revert to using original variable, $ushscript_dump. It is already defined in JOBSPROC_DUMP_MONITOR.